### PR TITLE
Add custom dispatcher to error-prototype

### PR DESCRIPTION
**Pull request by orkvi**
JerryScript-DCO-1.0-Signed-off-by: Orkenyi Virag orkvi@inf.u-szeged.hu

### CODE REVIEW

File: jerry-core/ecma/builtin-objects/ecma-builtin-error-prototype.c [hunk 1]
```diff
@@ -131,6 +145,38 @@ ecma_builtin_error_prototype_object_to_string (ecma_value_t this_arg) /**< this
...
+  */
+ecma_value_t
+ecma_builtin_error_prototype_dispatch_routine (uint8_t builtin_routine_id, /**< built-in wide routine
+                                                                                      *   identifier */
+                                                         ecma_value_t this_arg, /**< 'this' argument value */
+                                                         const ecma_value_t arguments_list_p[], /**< list of arguments
+                                                                                                 *   passed to
+                                                                                                 *   routine */
+                                                         uint32_t arguments_number) /**< length of arguments'
+                                                                                     *   list */
```
Comment (rerobika) [inline jerry-core/ecma/builtin-objects/ecma-builtin-error-prototype.c:162]: misaligned parameters

File: jerry-core/ecma/builtin-objects/ecma-builtin-error-prototype.c [hunk 2]
```diff
@@ -131,6 +145,38 @@ ecma_builtin_error_prototype_object_to_string (ecma_value_t this_arg) /**< this
...
+  *
+  * @return ecma value
+  *         Returned value must be freed with ecma_free_value.
+  */
+ecma_value_t
+ecma_builtin_error_prototype_dispatch_routine (uint8_t builtin_routine_id, /**< built-in wide routine
+                                                                                      *   identifier */
+                                                         ecma_value_t this_arg, /**< 'this' argument value */
+                                                         const ecma_value_t arguments_list_p[], /**< list of arguments
+                                                                                                 *   passed to
```
Comment (zherczeg) [inline jerry-core/ecma/builtin-objects/ecma-builtin-error-prototype.c:159]: No need to split a comment to 3 lines when there is enough space. Try to reduce these.

**Review by zherczeg — CHANGES_REQUESTED** [review]

File: jerry-core/ecma/builtin-objects/ecma-builtin-error-prototype.c [hunk 3]
```diff
@@ -131,6 +145,36 @@ ecma_builtin_error_prototype_object_to_string (ecma_value_t this_arg) /**< this
   return ecma_make_string_value (ecma_stringbuilder_finalize (&builder));
 } /* ecma_builtin_error_prototype_object_to_string */
 
+/**
+  * Dispatcher of the Generator built-in's routines
```
Comment (rerobika) [inline jerry-core/ecma/builtin-objects/ecma-builtin-error-prototype.c:149]: misaligned comment block, also it's not Generator builtin object.

**Review by rerobika — CHANGES_REQUESTED** [review]

**Review by galpeter — APPROVED** [review]
lgtm

**Review by zherczeg — APPROVED** [review]
LGTM

**Review by rerobika — APPROVED** [review]

**Pull request merged**